src/parsing.rs

```diff
@@ -23,6 +23,7 @@ enum ControlOperator {
 }
 
 pub const SCOPE_MARKER: &str = "_CDDL_CODEGEN_SCOPE_MARKER_";
+pub const EXTERN_DEPS_DIR: &str = "_CDDL_CODEGEN_EXTERN_DEPS_DIR_";
```

Comment:
Can we not easily have a test for this one?

Reply:
I guess we could but we'd have to pull in a dependency. I guess we could have a simple crate somewhere in the tests folder and use that and add it to the Cargo.toml of the test after generation as a local dependency. I'll try doing that.

Reply:
I added a test. It was a bit more work than I expected. It ended up also testing `--common-import-override` necessarily so that's a plus at least. I also fixed the failing tests (was only happening on github). Turns out after running cargo from a clean install would cause `cddl` to update to `0.9.4` (locally it was still using `0.9.1.` as specified  for me still) which broke basic group support parsing, at least in one of the ways it was used in some of the tests. I made an issue for it anweiss/cddl/#222 and set it to `=0.9.1` to get around that.